test/e2e/utils/verification/scylladbcluster/cql.go

```diff
@@ -4,18 +4,19 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"reflect"
+	"sort"
+	"strings"
+	"time"
+
 	"github.com/scylladb/scylla-operator/pkg/api/scylla/v1alpha1"
 	"github.com/scylladb/scylla-operator/pkg/helpers/slices"
 	"github.com/scylladb/scylla-operator/pkg/naming"
 	"github.com/scylladb/scylla-operator/test/e2e/framework"
 	v1alpha2 "github.com/scylladb/scylla-operator/test/e2e/utils/v1alpha1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
nit: metav1 (we can also change `v2` to `corev1` while we're at it)

Reply:
Yeah, we can configure it in a separate PR. :+1: